backend/src/openbeheer/api/views.py

```diff
@@ -158,18 +159,26 @@ def to_ob_field(name: str, annotation: type) -> OBField:
         attrs = get_annotations(self.data_type)
         return [to_ob_field(field, annotation) for field, annotation in attrs.items()]
 
-    def parse_query_params(self, request: Request) -> P:
+    def parse_query_params(self, request: Request, api_client: APIClient) -> P:
         "Parse incoming query parameters into a value of `self.query_type`"
-        params = convert(request.query_params, self.query_type)
+
+        # Sanitize params to prevent list values.
+        # TODO: Figure out how to deal with intentional lists?
+        params_dict = {
+            key: request.query_params.get(key) for key in request.query_params
```

Comment:
I think we need even more processing here as `page` is specified as `int` but query params end up a `str`.

Reply:
Already handled, see test cases